src/dir.proj

```diff
@@ -0,0 +1,292 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="dir.props" />
+  <Import Project="..\dir.targets" />
+
+  <PropertyGroup>
+    <BuildTargets>
+      BuildProjectsForNuGetPackages;
+      PublishSharedFrameworkAndSharedHost;
+      GenerateRuntimeGraph;
+      CopyHostArtifactsToSharedFramework;
+      CrossGenDirectory;
+      GenerateVersionFile;
+    </BuildTargets>
+  </PropertyGroup>
+
+  <Target Name="Build" DependsOnTargets="$(BuildTargets)" />
+
+  <UsingTask TaskName="GetBuildArgsByFrameworks" AssemblyFile="$(BuildToolsTaskDir)core-setup.tasks.dll"/>
+
+  <Target Name="BuildProjectsForNuGetPackages">
+    
+    <PropertyGroup>
+      <pOutDir>$(IntermediateOutputRootPath)forPackaging</pOutDir>
+    </PropertyGroup>
+    
+    <MakeDir  Condition="!Exists('$(pOutDir)')" Directories="$(pOutDir)" /> 
+
+    <PropertyGroup>
+      <Args>--build-base-path $(pOutDir) --configuration $(Configuration)</Args>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageProjects Include="Microsoft.DotNet.PlatformAbstractions/project.json" />
+      <PackageProjects Include="Microsoft.Extensions.DependencyModel/project.json" />
+    </ItemGroup>
+
+    <GetBuildArgsByFrameworks
+      OSGroup="$(OSGroup)"
+      ProjectJsonPaths="@(PackageProjects)">
+      <Output ItemName="buildCmdArgs" TaskParameter="BuildArgs" />
+    </GetBuildArgsByFrameworks>
+
+    <Exec Command="$(DotnetToolCommand) build $(Args) %(buildCmdArgs.Identity)" />
```

Comment:
I spoke with Peter P, and he mentioned that the reason we are doing this build and pack separately is for signing.  How do these assemblies get signed correctly?